test/cli.test.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
```

Comment:
:+1: :+1: the restructuring you've done here is awesome and the tests look great. The one last thing I would ask is that your assertions all have messages -- `assert.equal(found, expected, message)`.

I find it really difficult to track down what has actually failed in a set of tests where assertions don't have customized messages.